packages/mdc-select/component.ts

```diff
@@ -465,10 +467,10 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
     };
   }
 
-  private setEnhancedSelectedIndex_(index: number) {
+  private setSelectedIndex_(index: number) {
```

Comment:
We can potentially split this adapter method into smaller adapter methods. `setSelectedIndex` adapter method seems to be doing many DOM operations which is again harder to implement by Frameworks. This also makes harder to support business logic of `setSelectedIndex` adapter method that are internal to Foundation.

WDYT?

Reply:
Great idea - done.